### ISSUE TITLE
Don't close the PDF if it is deleted.

### ISSUE THREAD

**Issue by NicoJG** [tomoki1207/vscode-pdfviewer#135]
In my workflow, I frequently update PDFs, and I love the auto-update feature of this extension. 
However, some tools I use actually remove the PDF file or even the directory it is in. 
When this happens, the PDF viewer instantly closes. 
Is there a way to keep the PDF viewer open even if the file was deleted?
And additionally keep the auto-update running, e.g. update the PDF viewer when the file is created again.

**Comment by flying-sheep** [comment]
Yeah this is vital. I love this extension but for TeX projects, it’s unusable because of this.

**Comment by flying-sheep** [comment]
See also #125
